src/esp/batched_sim/EpisodeGenerator.h

```diff
@@ -13,7 +13,9 @@ namespace batched_sim {
 struct EpisodeGeneratorConfig {
   int numEpisodes = 100;
   int seed = 3; // this is 3 for legacy reason
-  int numStageVariations = 12; // see selectedReplicaCadBakedStages
+  // int numStageVariations = 12; // see selectedReplicaCadBakedStages
+  int minStageNumber = 0;
+  int maxStageNumber = 12;
```

Comment:
off-by-one here? previously, the max stage index was 11, because there are only 12 stages.